prow/bugzilla/types.go

```diff
@@ -107,6 +109,81 @@ type Bug struct {
 	Whiteboard string `json:"whiteboard,omitempty"`
 }
 
+// BugCreate holds the info needed to create a new bug
```

Comment:
I know I didn't put every possible JSON field for a GetBug call into the Bug struct -- but we don't need any of the ones I missed for create? Nice.